i18n/cs.po

```diff
@@ -20,6 +21,10 @@ msgstr "Mapa bez názvu, {{date}}"
 
 msgid "Map \"{{- name}}\" is saved."
 msgstr "Mapa \"{{- name}}\" je uložena."
+msgstr ""
```

Comment:
This looks like one too many `msgstr` for this string, right?

Reply:
This might cause the file to be corrupt. Suggest rebasing this branch and dropping the commit with the translations

Reply:
Well spotted, thanks. I mustn't have resolved the conflicts correctly during my previous rebase. Rebasing again and dropping this commit seems to have worked fine. I'm not completely sure how it ended up in there in the first place.